src/components/LoadingComments/LoadingComments.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { LoadingComments } from "./LoadingComments";
+
+export default { component: LoadingComments, title: "LoadingComments" };
+
+export const Default = () => {
+  return (
+    <>
```

Comment:
no need for fragments
```suggestion
export const Default = () => <LoadingComments />
```